src/main/java/org/opensrp/web/rest/ClientFormResource.java

```diff
@@ -0,0 +1,167 @@
+package org.opensrp.web.rest;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.entity.ContentType;
+import org.apache.http.util.TextUtils;
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+import org.opensrp.domain.IdVersionTuple;
+import org.opensrp.domain.postgres.ClientForm;
+import org.opensrp.domain.postgres.ClientFormMetadata;
+import org.opensrp.service.ClientFormService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.List;
+
+@Controller
+@RequestMapping(value = "/rest/clientForm")
+public class ClientFormResource {
+
+    private static Logger logger = LoggerFactory.getLogger(EventResource.class.toString());
+    protected ObjectMapper objectMapper;
+    private ClientFormService clientFormService;
+
+    @Autowired
+    public void setClientFormService(ClientFormService clientFormService) {
+        this.clientFormService = clientFormService;
+    }
+
+    @Autowired
+    public void setObjectMapper(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    @RequestMapping(method = RequestMethod.GET)
+    @ResponseBody
```

Comment:
This is not required. Its usually put  in the method parameter to bind the body of the request to pojo or string